docs/examples/config

```diff
@@ -13,6 +13,7 @@ poll_method		epoll		# poll, select, epoll ..
 sip_cafile		/etc/ssl/certs/ca-certificates.crt
 #sip_trans_def		udp
 sip_verify_server	yes
+sip_tos			104
```

Comment:
is TOS for SIP really needed ?

TOS/QoS is mainly about the real-time media traffic that gets higher priority in a network.

Reply:
I think so. It seems that different manufacturer also set it for SIP with different default values. https://kb.smartvox.co.uk/ip-phones/sip-configuration/voip-qos-settings-part-2/

We need it also for our products.